File: 이유진/codes/Lv.1/약수의개수와덧셈.java
class Solution {
    public int solution(int left, int right) {
        int answer = 0;
        int num = left;
        while(num <=  right){
            int n = 0;
            for(int i = 1; i <= num; i++){
                if(num%i == 0) n++;
            }
            if(n%2 == 0) answer += num;
            if(n%2 != 0) answer -= num;
            num++;
        }
        return answer;
    }
}

File: 조선빈/Lv.1/약수의 개수와 덧셈.py
def measure(num):
    cnt = 0
    for i in range(1, num+1):
        if num%i == 0:
            cnt += 1
    return cnt

def num_measure(m_left,left):
    answer = 0
    if m_left%2 == 0:
        answer = answer + left
    else:
        answer = answer - left
    return answer

def solution(left, right):
    answer = 0
    for i in range(left,right+1):
        measure(i)
        answer = answer + num_measure(measure(i),i)
    return answer
